dataland-frontend/package.json

```diff
@@ -21,51 +21,51 @@
     "checkdependencies": "depcheck --ignores='@clients/*,@e2e/*,@sharedUtils/*,@ct/*,@vue/tsconfig,axios,tsconfig-paths,eslint-plugin-*,@vue/eslint-config-*'"
   },
   "dependencies": {
-    "@formkit/core": "^1.5.3",
-    "@formkit/themes": "^1.5.3",
-    "@formkit/vue": "^1.5.3",
+    "@formkit/core": "^1.5.9",
+    "@formkit/themes": "^1.5.9",
+    "@formkit/vue": "^1.5.9",
     "@ibm/plex": "^6.4.0",
     "axios": "^1.6.7",
     "currency-codes": "^2.1.0",
-    "i18n-iso-countries": "^7.8.0",
-    "keycloak-js": "^22.0.5",
+    "i18n-iso-countries": "^7.10.1",
+    "keycloak-js": "^24.0.1",
```

Comment:
What about the update of the keycloak docker image version?
It is still on 22. keycloak-js is now on 24. Might that not lead to incompatibilities if the API was changed? We probably would have noticed in CI.

Reply:
correct. i updated the keycloak version in the dockerfile